modules/gitops-release-notes-1-11-5.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
```

Comment:
Not a big deal AFAIA, but the space is not necessary. Best to keep consistent with the example in the doc_guidelines.adoc file.

Reply:
Thanks for letting me know about this @dfitzmau. I will implement it in the future as well. Fixed the feedback now.